core/src/main/java/org/apache/accumulo/core/clientImpl/ClientContext.java

```diff
@@ -918,21 +918,26 @@ public AuthenticationToken token() {
   @Override
   public synchronized void close() {
     if (closed.compareAndSet(false, true)) {
-      if (zooCacheCreated.get()) {
-        zooCache.get().close();
-      }
-      if (zooKeeperOpened.get()) {
-        zooSession.get().close();
-      }
       if (thriftTransportPool != null) {
+        log.info("Closing Thrift Transport Pool");
         thriftTransportPool.shutdown();
       }
       if (scannerReadaheadPool != null) {
+        log.info("Closing Scanner ReadAhead Pool");
         scannerReadaheadPool.shutdownNow(); // abort all tasks, client is shutting down
       }
       if (cleanupThreadPool != null) {
+        log.info("Closing Cleanup ThreadPool");
         cleanupThreadPool.shutdown(); // wait for shutdown tasks to execute
       }
+      if (zooCacheCreated.get()) {
+        log.info("Closing ZooCache");
```

Comment:
I think these log messages could be at debug. They're pretty noisy, and it's not like we log at INFO when we create them. They are all internal resources. The user doesn't necessarily care to see all this internal thread cleanup for the normal case.

Reply:
Changed to debug in d41dbcc